charts/cluster-autoscaler/Chart.yaml

```diff
@@ -11,4 +11,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.28.0
+version: 9.29.0
```

Comment:
This will need to be bumped again as https://github.com/kubernetes/autoscaler/pull/5768 got merged, just a patch bump should do as we'll just be doing docs bump here.

Reply:
addressed the above comment.